pmoves/services/hi-rag-gateway-v2/routes/health.py

```
@@ -0,0 +1,75 @@
+    _cfg._rerank_model_label = label.strip() or None
+    return {"ok": True, "rerank_model_label": _cfg._rerank_model_label}
+
+
+@router.post("/hirag/admin/refresh")
+def hirag_admin_refresh(_=Depends(require_admin_tailscale)):
+    refresh_warm_dictionary()
+    return {"ok": True, "last_refresh": _warm_last}
+
+
+@router.post("/hirag/admin/cache/clear")
+def hirag_admin_cache_clear(_=Depends(require_admin_tailscale)):
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)